src/coreComponents/physicsSolvers/contact/SolidMechanicsEFEMKernelsBase.hpp

```diff
@@ -249,6 +254,15 @@ class EFEMKernelsBase :
     // Gauss contribution to Kww, Kwu and Kuw blocks
     real64 Kww_gauss[3][3], Kwu_gauss[3][nUdof], Kuw_gauss[nUdof][3];
 
+    // Gauss contirbution to eqMStress which is EqMatrix*effStress, all stresses are in Voigt notation
+    real64 eqMStress_gauss[3]{};
+    real64 effStress[6] = {m_effStress[k][q][0], 
```

Comment:
if it's already in Voigt notation I think that you can directly use:

```c++
LvArray::tensorOps::Ri_eq_AijBj<3, 6> (eqMStress_gauss, eqMatrix, m_effStress[k][q]);
```  
coz those `LvArray` functions also work with `arraySlice`